libbeat/version/helper.go

```diff
@@ -32,6 +32,9 @@ var (
 	buildTime = "unknown"
 	commit    = "unknown"
 	qualifier = ""
+
+	// OSS provides if this is the OSS distro
+	OSS = true
```

Comment:
the global with the `init` override from the xpack root command is kind of sick, but I'm not sure there's a simpler/cleaner way of doing this